gst/tensor_ros_src/tensor_ros_src.h

```diff
@@ -53,7 +53,9 @@ struct _GstTensorRosSrc
 {
   GstPushSrc parent;
 
-  GstCaps *caps;
+  GstCaps *caps;          /** Output caps */
```

Comment:
is it needed to store this caps?

Reply:
If `gst_pad_get_current_caps()` is used, then this variable could be removed. I will.